client/src/components/RelatedProducts/Cards/Individualcard.jsx

```diff
@@ -1,21 +1,23 @@
-import React, { useContext } from 'react';
+import React, {
+  createContext, useContext, useEffect, useState,
+} from 'react';
 import styled from 'styled-components';
-import sampleData from '../../sampleData';
+
 
 // should retrieve all related products from initial product page
 // map each individual card with all the information
 
-export default function Individualcard() {
+export default function Individualcard({ product }) {
   return (
     <div>
       <CardText>
-        <p>{sampleData.products[1].category}</p>
```

Comment:
consider using spread operating before passing the variable in. it's something like const {category, name } = sampleData.products[i]